__tests__/app.test.js

```diff
@@ -59,7 +59,8 @@ describe('/api/articles/:article_id endpoint', () => {
       .get('/api/articles/1')
       .expect(200)
       .then(({ body }) => {
-        expect(body.article).toEqual(expectedArticle)
```

Comment:
Ideally in future, to ensure this does not occur, you may choose to use matchers such as expect.objectContaining or .toMatchObject and check the type (if it is something that could potentially change)